conformance/lang/expressions/shift-expression/left_shift.balt

```diff
@@ -0,0 +1,3816 @@
+    //                     = Equivalent binary representation (basic type): 00000000 00000000 00000000 00000000 00000000 00000000 00000000 11111111
+    // Shift amount = Before applying mask, In binary: 11111111, In decimal: 255
+    //                After applying mask, In binary: 111111, In decimal: 63
+    // Result = In binary: 10000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
+    //        = In decimal: (-9223372036854775808)
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int:Unsigned8' as the LHS operand and 'int' as the RHS
+             operand.
+Labels: DecimalNumber, float, int, int:Unsigned8, shift-expr, unary-minus
+
+function errorFunction(int:Unsigned8 x, int a) {
+    int:Unsigned8 _ = x << a; // @error expected 'int:Unsigned8' but found 'int'
+    float _ = x << -123; // @error expected 'float' but found 'int'
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int:Unsigned8' as the LHS operand and builtin subtypes of
+             integer as the RHS operand.
+Labels: decimal, float, int:Signed16, int:Signed32, int:Signed8, int:Unsigned16, int:Unsigned32, int:Unsigned8,
+        shift-expr
+
+function errorFunction(int:Unsigned8 x, int:Signed8 a, int:Unsigned8 b, int:Signed16 c, int:Unsigned16 d,
+                        int:Signed32 e, int:Unsigned32 f) {
+    int:Signed16 _ = x << a; // @error expected 'int:Signed16' but found 'int'
+    decimal _ = x << b; // @error expected 'decimal' but found 'int'
+    int:Unsigned8 _ = x << c; // @error expected 'int:Unsigned8' but found 'int'
+    int:Signed16 _ = x << d; // @error expected 'int:Signed16' but found 'int'
+    float _ = x << e; // @error expected 'float' but found 'int'
+    int:Unsigned16 _ = x << f; // @error expected 'int:Unsigned16' but found 'int'
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int:Unsigned8' as the LHS operand and byte as the RHS
+             operand.
+Labels: byte, int:Unsigned8, shift-expr, var
+
+function errorFunction(int:Unsigned8 x, byte a) {
+    var x1 = x << a;
+    int:Unsigned8 _ = x1; // @error expected 'int:Unsigned8' but found 'int'
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int:Unsigned8' as the LHS operand and user-defined subtype
+             of integer as the RHS operand.
+Labels: DecimalNumber, int:Unsigned8, shift-expr, type, union-type, unary-minus
+
+type Ints -1|2|-128;
+
+function errorFunction(int:Unsigned8 x, Ints a) {
+    int:Unsigned8 _ = x << a; // @error expected 'int:Unsigned8' but found 'int'
+    Ints _ = x << a; // @error expected 'Ints' but found 'int'
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int:Unsigned8' as the LHS operand and optional 'int' as
+             the RHS operand.
+Labels: int, int:Unsigned8, optional-type, shift-expr, var
```

Comment:
```suggestion
Labels: int:Unsigned8, optional-type, shift-expr, var
```

Reply:
Type descriptor `int` is already used within this test.